### PR TITLE
[Ldap] Add security LdapUser and provider (4.4)

### PR DESCRIPTION
**Pull request by hiddewie**
#12088 and symfony/symfony#32824

image

### CODE REVIEW

File: security/ldap.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Symfony provides different means to work with an LDAP server.
 The Security component offers:
 
 * The ``ldap`` :doc:`user provider</security/user_provider>`, using the
-  :class:`Symfony\\Component\\Security\\Core\\User\\LdapUserProvider`
+  :class:`Symfony\\Component\\Ldap\\Security\\LdapUserProvider`
```
Comment (OskarStark) [inline security/ldap.rst]: Please remove the spaces at the beginning like before

Comment (hiddewie) [reply security/ldap.rst]: Which spaces? Only the classname has been updated in this PR.

Comment (OskarStark) [reply security/ldap.rst]: Weird on a phone it looks like there were spaces at the beginning. Sorry

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Hidde.

**Pull request merged**